app/src/main/java/com/eveningoutpost/dexdrip/GcmListenerSvc.java

```diff
@@ -542,8 +544,12 @@ public void run() {
                 } else if (action.equals("ssom")) {
                     if (Home.get_master()) {
                         if (payload.equals("challenge string")) {
-                            UserError.Log.e(TAG, "Stopping sensor by remote");
-                            StopSensor.stop();
+                            if (Pref.getBoolean("plus_accept_follower_actions", true)) {
+                                UserError.Log.e(TAG, "Stopping sensor by remote");
+                                StopSensor.stop();
+                            } else {
+                                UserError.Log.wtf(TAG, "Stop sensor by follower rejected because follower actions are disabled");
```

Comment:
Is this really a WTF? WTFH (What a terrible failure) is more severe than error log. The error which should never ever happen. It may force the app to hold for writing the logs before termination.

Reply:
What are the alternatives?  
I just copied the format from another adjacent log, "Challenge string failed for ssom".

Reply:
I thought more about this.  This log is generated only when there is a disagreement between a follower and the master.    Either the master is a kid who needs to enable the switch to allow the parent to control, or the master is a grown-up who needs to change the security code to disable an idiot from following.
Either way, this is serious situation that requires immediate attention.  So, I would leave it as wtf.

Reply:
But there is no notification about this. It is simply a log entry at the collectors side. Neither the sensor-wearing user nor the remote controller knows about the stop failure.

log.wtf is not a categorization at the user level. It is a categorization from the point of program flow and in this case it is not a terrible failure. Please change it to `UserError.Log.e()` or better to `UserError.Log.w()`.

Reply:
Done